### ISSUE TITLE
Allow for game to be free but require an email address

### ISSUE THREAD

**Issue by initials** [itchio/itch.io#19]
One of the most powerful things you can have is a list of people who have played your game, or potential fans.

It would be great to allow a game to be free in exchange for an email address, so that they can be contacted when future releases are out.

For reference, Bandcamp does this well.

**Comment by Guard13007** [comment]
Wouldn't this also be nice so that an account can "buy" a free game that later on will have a price? That way it is in their library even if it gets a price later.

**Comment by fasterthanlime** [comment]
@Guard13007 true!

Although I personally think that feature should be opt-out for downloaders. It might be important for a section of the community to be able to download free games (relatively) anonymously.

**Comment by Guard13007** [comment]
> Although I personally think that feature should be opt-out for downloaders

Wouldn't opt-in be better? People don't often read the text, just the big buttons. I've had this problem myself with people thinking they had to buy a game I put on Itch.io and me having to explain to them to click "No thanks, just take me to the downloads." if they didn't want to pay a dollar.

**Comment by aaronfranke** [comment]
> Wouldn't this also be nice so that an account can "buy" a free game that later on will have a price? That way it is in their library even if it gets a price later.

I disagree with this. I like Itch's current approach, of allowing people to donate a small amount of money (>$0.5 last I checked) to add a free game to your account. It encourages people to donate, for the sake of convenience, but people who don't want to donate can still download the game.
